packages/ui/Scroller/Scroller.module.less

```diff
@@ -1,5 +1,9 @@
-.hideNativeScrollbar {
-	&::-webkit-scrollbar {
-		display: none;
-	}
-}
+.willChange {
+	will-change: transform;
+}
+
+.hideNativeScrollbar {
+	&::-webkit-scrollbar {
+		display: none;
+	}
+}
```

Comment:
As we discussed , let's combine two CSS class selector above into one selector like `.scroller`.